x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.test.tsx

```diff
@@ -90,27 +101,30 @@ const { loadRuleAggregationsWithKueryFilter } = jest.requireMock(
 const { loadRulesWithKueryFilter } = jest.requireMock('../../../lib/rule_api/rules_kuery_filter');
 const { loadActionTypes, loadAllActions } = jest.requireMock('../../../lib/action_connector_api');
 
-const actionTypeRegistry = actionTypeRegistryMock.create();
-const ruleTypeRegistry = ruleTypeRegistryMock.create();
-
-ruleTypeRegistry.list.mockReturnValue([ruleType]);
-actionTypeRegistry.list.mockReturnValue([]);
-
 const useKibanaMock = useKibana as jest.Mocked<typeof useKibana>;
-
-beforeEach(() => {
-  (getIsExperimentalFeatureEnabled as jest.Mock<any, any>).mockImplementation(() => false);
-});
-
-// This entire test suite is flaky/timing out and has been skipped.
-afterEach(() => {
-  jest.clearAllMocks();
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+      cacheTime: 0,
+    },
+  },
 });
 
 // FLAKY: https://github.com/elastic/kibana/issues/134922
 // FLAKY: https://github.com/elastic/kibana/issues/134923
 // FLAKY: https://github.com/elastic/kibana/issues/134924
 
+const AllTheProviders = ({ children }: { children: any }) => (
+  <IntlProvider locale="en">
+    <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+  </IntlProvider>
+);
+
+const renderWithProviders = (ui: any) => {
+  return render(ui, { wrapper: AllTheProviders });
+};
+
 describe.skip('Update Api Key', () => {
```

Comment:
should we try to enable this as well? :sweat_smile: 

not a huge deal, if not we can always do another ticket/PR

Reply:
test fixed here https://github.com/elastic/kibana/pull/147014/commits/7b0333d7ebb54192ece897bb935beb116f264a61